watch-and-repack.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Relative paths to the web repositories
+PENUMBRA_ZONE_WEB_PATH="../../web"
+PRAX_REPO_PATH="."
+
+# Ensure 'pack-public.sh' is executable
+chmod +x "$PENUMBRA_ZONE_WEB_PATH/pack-public.sh"
```

Comment:
don't modify permissions like this

Reply:
script can remove this, and dev can manually set the permission. I just did this for my personal convenience